ecommerce/apps.py

```
@@ -0,0 +1,6 @@
+from django.apps import AppConfig
+
+
+class EcommerceConfig(AppConfig):
+    default_auto_field = 'django.db.models.BigAutoField'
```

Ruff (0.8.2)

Single quotes found but double quotes preferred

Replace single quotes with double quotes

(Q000)